src/components/calculator.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './calc.css';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="mainContainer">
+        <div className="displayInputs" />
+        <div className="calcDiv">
+          <button type="button" className="btn">AC</button>
+          <button type="button" className="btn">+/-</button>
+          <button type="button" className="btn">%</button>
+          <button type="button" className="btn btnaction">/</button>
+          <button type="button" className="btn">7</button>
+          <button type="button" className="btn">8</button>
+          <button type="button" className="btn">9</button>
+          <button type="button" className="btn btnaction">X</button>
+          <button type="button" className="btn">4</button>
+          <button type="button" className="btn">5</button>
+          <button type="button" className="btn">6</button>
+          <button type="button" className="btn btnaction">-</button>
+          <button type="button" className="btn">1</button>
+          <button type="button" className="btn">2</button>
+          <button type="button" className="btn">3</button>
+          <button type="button" className="btn btnaction">+</button>
```

Comment:
- [ ] (OPTIONAL) Great job so far 👍 To match the design exactly kindly add the `0` to the first input as in the template. Also, you can research on Google on how to input the multiplication (`×`) and division (`÷`) signs 💯